nomos-core/src/block/builder.rs

```diff
@@ -90,13 +91,13 @@ where
     }
 
     #[must_use]
-    pub fn with_blobs(mut self, blobs: impl Iterator<Item = B> + 'static) -> Self {
+    pub fn with_blobs(mut self, blobs: impl Iterator<Item = C> + 'static) -> Self {
```

Comment:
how about naming this as `with_blob_certificates` for clarity? also the name of parameter.

Reply:
Yes, missed it! Good catch!